### PR TITLE
update README.md on how to add pull secret

### PR DESCRIPTION
**Pull request by jianzzha**
**Document on how to pull images from private repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,29 @@ MicroShift keeps all its state in its data-dir, which defaults to `/var/lib/micr
...
+
+MicroShift may not have the pull secret for the registry that you are trying to use. For example, MicroShift does not have the pull secret for registry.redhat.io. In order to use this registry, there are several approaches. The first approach is to use podman login,
+```sh
+podman login registry.redhat.io
+```
+
+Once the podman login is complete, MicroShift will be able to pull images from this registry. This approach works across name spaces.
+
+This approach assumes podman is installed. This might not be true for all MicroShift environments. For example, if MicroShift is installed through RPM, CRI-O will be installed as dependency, but no podman. In this case, one can choose to install podman separately, or use other approaches described below. 
+
```
Comment (sallyom) [inline README.md:139]: * edit `as dependency` to `as a dependency` and  `but no podman` to `but not podman` 
* wdyt, is this more clear? `In this case, one can choose to install podman separate from the rpm and then login as above, or, to allow CRI-O access to private registries, create a pull secret as described below`

File: README.md [hunk 2]
```diff
@@ -126,6 +126,29 @@ MicroShift keeps all its state in its data-dir, which defaults to `/var/lib/micr
...
+The second approach is to create a pull secret, then let the service account to use this pull secret. This approach works within a name space. For example, if the pull secret is stored in a json formatted file "secret.json",
+```sh
```
Comment (sallyom) [inline README.md:141]: edit `then link the secret to a service account in the namespace` (or, is it always `the default service account`? if so, `then link the secret to the default service account in the namespace`)

File: README.md [hunk 3]
```diff
@@ -126,6 +126,29 @@ MicroShift keeps all its state in its data-dir, which defaults to `/var/lib/micr
...
 
+### Pulling Container Image From Private Registries
+
+MicroShift may not have the pull secret for the registry that you are trying to use. For example, MicroShift does not have the pull secret for registry.redhat.io. In order to use this registry, there are several approaches. The first approach is to use podman login,
+```sh
+podman login registry.redhat.io
+```
+
+Once the podman login is complete, MicroShift will be able to pull images from this registry. This approach works across name spaces.
+
```
Comment (sallyom) [inline README.md:137]: name spaces / namespaces

**Review by sallyom — COMMENTED** [review]
a few nits, this is good information!

**Comment by sallyom** [comment]
> > > Can you add this as well to docs/index.md so that it's reflected on microshift.io?
> > 
> > 
> > We should probably add this type of content on the website part, and link from the readme.md here to the website to avoid maintaining stuff in two places.
> 
> Sure. Any pointer on how to update the content on the website?

hm, we shouldn't have that content in 2 places. I think we need to link the docs in the README.md, but for now, add your information to the docs index.md. The docs are new, and now that they are there that's where we'll direct people rather than the README (ie, I think we'll update the README to have a brief intro then link to the docs)

**Comment by iranzo** [comment]
Handled on https://github.com/redhat-et/microshift-documentation/pull/9 respecting authorship of commit

**Comment by cooktheryan** [comment]
closed as it was merged in the docs repo

**Pull request closed without merging**